### ISSUE TITLE
optional autoplay

### ISSUE THREAD

**Issue by florianGierlichs** [ghermens/media2click#4]
Hi,
when I click the first time and the iFrame loaded, I have the problem, that the youtube play-button to start the video has no visual focus state. This leads to bad accessibility for keyboard users.
I couldn't find a way, to style the button by myself via CSS or JS.
So I thought about an option to set the autoplay functionality. This of course would conflict with the "2click" idea, but it would improve the accessibility and users doesn't have to guess their focused element.

![image](https://user-images.githubusercontent.com/58342773/105856914-07a26000-5fea-11eb-9035-de6d7217e3df.png)

**Comment by ghermens** [comment]
Hi Florian,

> when I click the first time and the iFrame loaded, I have the problem, that the youtube play-button to start the video has no visual focus state. This leads to bad accessibility for keyboard users.

when I click the placeholder, focus is transferred to the newly loaded iFrame. I then need another 3 "tab"s on the keyboard to move the focus to the play button. There's a blue outline on the button if focused.

> I couldn't find a way, to style the button by myself via CSS or JS.

Afair there is no way to influence the iFrame content with JS or CSS from the outside.

> So I thought about an option to set the autoplay functionality. This of course would conflict with the "2click" idea, but it would improve the accessibility and users doesn't have to guess their focused element.

You can set the video to autoplay in the media content element as recommended in the [manual](https://docs.typo3.org/p/amazing/media2click/1.1/en-us/User/Index.html#video). This will save the user the second click without jeopardizing privacy.
Of course you should only do so if you don't allow permanent activation ...
